[DOC-97] Added ycqlsh commands: SERIAL, TRACING, and UNICODE. (#31226)
yugabyte/yugabyte-db: docs/content/stable/explore/ycql-language/keyspaces-tables.md

## Unsupported cqlsh commands

> Some Apache Cassandra `cqlsh` commands differ or are not applicable on YugabyteDB. Examples:

Reviewer: ```suggestion
Some Apache Cassandra cqlsh commands differ or are not applicable on YugabyteDB. This includes the following:
```
